sources/odoo.conf

```diff
@@ -1,10 +1,9 @@
 [options]
 
 
-; --------------------
```

Comment:
Sorry I meant remove the 3 lines (since there's no need to change the password anymore) ;)